pkg/services/featuremgmt/registry.go

```diff
@@ -766,5 +766,12 @@ var (
 			Stage:        FeatureStageExperimental,
 			Owner:        grafanaPluginsPlatformSquad,
 		},
+		{
+			Name:         "alertingSaveStateAsync",
+			Description:  "Writes the state only on a given interval to database",
```

Comment:
[nit] I'd call this something like period rather than async, as though the writing is async, the more interesting characteristic arguably is that it's periodic.

```suggestion
			Name:         "alertingSaveStatePeriodic",
			Description:  "Writes the state periodically to the database, asynchronous to rule evaluation",
```